src/test/shell/bazel/testdata/exec_wasm.wat

```diff
@@ -0,0 +1,133 @@
+(module
+  (memory 1)
+
+  (func $allocate (export "allocate")
+    (param $size i32)
+    (param $align i32)
+    (result i32)
+    (local $orig_page_count i32)
+
+    ;; Pass each allocation directly to `memory.grow`, which allocates the
+    ;; requested number of pages (each page is 65535 bytes).
```

Comment:
The wasm spec seems to suggest this should be 65536 instead? https://webassembly.github.io/spec/core/exec/runtime.html#page-size

Reply:
Oops, that's embarrassing. Fixed.